dotnet/Gherkin/Gherkin.csproj

```
@@ -44,4 +44,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Cucumber.Messages" Version="28.*" />
```

```suggestion
    <PackageReference Include="Cucumber.Messages" Version="[27.0, 29.0)"/>
```

That way Renovate can automatically upgrade the upper bound.